CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [unreleased]
+
+### 🚀 Features
+
+- Add "elsewhere" option to fetch, pull and push menu
+- Syntax highlighting for Elixir
+
```

Comment:
Ah, I should've noted in my suggestion: this is generated when I make a release. So the commit [Add changelog entry](https://github.com/altsem/gitu/pull/160/commits/e45cd82595e89e2d38019c214dc3fdc865633acd) wouldn't be needed.

Reply:
oh, I already force-pushed to remove the commit

Reply:
:+1: Ah you're fast! My merge failed because of it. Let's include all the commits then :)